client/app/bundles/course/assessment/pages/AssessmentStatistics/LiveFeedbackHistory/LiveFeedbackDetails.tsx

```diff
@@ -73,10 +73,10 @@ const LiveFeedbackDetails: FC<Props> = (props) => {
           ref={editorRef}
           cursorStart={startingLineNum - 1}
           disabled
+          focus
           // This height matches the prompt height exactly so there is no awkward scroll bar
           // and the prompt does not expand weirdly when description is opened
-          focus
-          height="482px"
+          height="440px"
```

Comment:
please refrain from using pixel unless there's no other way to use other than pixel.. might be able to use rem instead.. See how we use EditorField in the past

Reply:
done